src/help.js

```diff
@@ -6,6 +6,7 @@ const commands = [
   'alias [-p] [name=[value]]',
   'cat [-AbeEnstTv] [files ...]',
   'cd [dir]',
+  'clear [files ...]',
```

Comment:
As above, there's no `[files ...]` params expected, so you can just make the usage `clear`. :)